extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveProcessor.java

```diff
@@ -415,19 +424,44 @@ private boolean hasAnnotation(MethodInfo method, short paramPosition, DotName an
                         .setDefaultProducesHandler(new DefaultProducesHandler.DelegatingDefaultProducesHandler(handlers));
             }
             serverEndpointIndexer = serverEndpointIndexerBuilder.build();
+            Map<String, List<List<EndpointConfig>>> allMethods = new HashMap<>();
 
             for (ClassInfo i : scannedResources.values()) {
                 if (!appResult.keepClass(i.name().toString())) {
                     continue;
                 }
                 ResourceClass endpoints = serverEndpointIndexer.createEndpoints(i);
+                List<ResourceMethod> resourceMethods = endpoints.getMethods();
+                getResourceMethodByPath(endpoints.getPath(),
```

Comment:
I have a very hard time reviewing this due to all the stream usage. Perhaps it's just my fault, though. What do you think @geoand ?

Reply:
I didn't want to be the person who beat that drum again...
I wanted first to see if someone else would raise this same concern (to ensure it wasn't just a weird personal taste of mine)... Since now you have, I'll just say it:

I dislike stream usage like this very much... I don't think it brings any value, but instead most of the time makes reading the code harder...

Reply:
ok, i didn't liked them at first, but now that I am used to it I find them more clear. Any way I am not religious about it, so I can refactor to classical loop.

Reply:
So the basic rule is no stream ?

Reply:
We don't have a hard rule (for build time code like this). But we really think that using streams in streams and then more streams does not make the code more readable, if anything it makes it less readable.